installer/scripts/omsadmin.sh

```diff
@@ -670,7 +670,12 @@ onboard()
             # Set up a cron job to run the OMSConsistencyInvoker every 15 minutes
             # This should be done regardless of MetaConfig creation
             if [ ! -f /etc/cron.d/OMSConsistencyInvoker ]; then
```

Comment:
[Not a problem, just documenting behavior for reference]

In VM extension scenario, upgrade will happen with the sequence `disable with omsadmin -> --remove bundle -> --upgrade bundle [no params] -> onboard with omsadmin`. Due to `--remove` the cron job will already be removed so this should satisfy extension upgrade scenario. And for manual `--upgrade` the necessary change is made to pre-remove the cron job.